Plugins/NativeServerPlugin/src/peer_conductor.cpp

```diff
@@ -256,14 +256,14 @@ void PeerConductor::HandlePeerMessage(const string& message)
 		if (type == "offer-loopback")
 		{
 			//TODO(bengreenier): reimplement
```

Comment:
Do we have an issue for this TODO item? Not sure what reimplement implies

Reply:
no. but we do now: https://github.com/3DStreamingToolkit/3DStreamingToolkit/issues/288